### DISCUSSION TITLE
Can build CRM with supabase?

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [supabase/supabase#22342] [Questions]
Our team want using supabase to build CRM with complex function. Is it feasible to develop a CRM (Customer Relationship Management) system using Supabase?

**Comment by jhb-dev** [comment]
I would say yes! Supabase provides a powerful backend with a PostgreSQL database, authentication, real-time capabilities, and serverless functions, making it a strong candidate for building a CRM system. 

For what reasons do you doubt it?
